.github/workflows/android.yml

```
@@ -0,0 +1,25 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Copy configuration files
+      run: cp google-services.json.sample app/google-services.json && cp keystore.properties.sample keystore.properties
+    - name: Build with Gradle
+      run: ./gradlew assembleDebug
+    - name: Run Tests
+      run: ./gradlew test
```

why not test before assembleDebug?
```suggestion
    - name: Run Tests
      run: ./gradlew test
    - name: Build with Gradle
      run: ./gradlew assembleDebug
```